receiver/memcachedreceiver/documentation.md

```diff
@@ -10,15 +10,15 @@ These are the metrics available for this scraper.
 | ---- | ----------- | ---- | ---- | ---------- |
 | memcached.bytes | Current number of bytes used by this server to store items. | By | Gauge | <ul> </ul> |
 | memcached.commands | Commands executed. | 1 | Sum | <ul> <li>command</li> </ul> |
-| memcached.current_connections | The current number of open connections. | connections | Sum | <ul> </ul> |
+| memcached.connections.current | The current number of open connections. | connections | Sum | <ul> </ul> |
+| memcached.connections.total | Total number of connections opened since the server started running. | connections | Sum | <ul> </ul> |
 | memcached.current_items | Number of items currently stored in the cache. | 1 | Sum | <ul> </ul> |
 | memcached.evictions | Cache item evictions. | 1 | Sum | <ul> </ul> |
 | memcached.network | Bytes transferred over the network. | by | Sum | <ul> <li>direction</li> </ul> |
 | memcached.operation_hit_ratio | Hit ratio for operations, expressed as a percentage value between 0.0 and 100.0. | % | Gauge | <ul> <li>operation</li> </ul> |
 | memcached.operations | Operation counts. | 1 | Sum | <ul> <li>type</li> <li>operation</li> </ul> |
-| memcached.rusage | Accumulated user and system time. | 1 | Sum | <ul> <li>state</li> </ul> |
+| memcached.rusage.cpu | Accumulated user and system time. | 1 | Sum | <ul> <li>state</li> </ul> |
```

Comment:
@djaglowski should this be "memcached.cpu.usage" ?

Reply:
Yes, you're right. 

@Mrod1598 ^